docs/running-as-firefox-panel.md

```diff
@@ -0,0 +1,98 @@
+# Running as the firefox panel
+
+* [Getting Started with Firefox Nightly](#getting-started-with-firefox-nightly)
+  * [For Windows Developers](#for-windows-developers)
+* [Configuring Firefox path in the debugger](#configuring-firefox-path-in-the-debugger)
+* [Watching for Changes](#watching-for-changes)
+* [Running Nightly with the local debugger](#running-nightly-with-the-local-debugger)
+* [Getting Help](#getting-help)
+
+## Getting Started with Firefox Nightly
+
+Running firefox is similar to running [Mochitests](./mochitests.md)
+
+**Requirements**
+
+* mercurial ( `brew install mercurial` )
+* autoconf213 ( `brew install autoconf@2.13 && brew unlink autoconf` )
+
+On the first run, this will download a local copy of Firefox and set up an [artifact build](https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/Build_Instructions/Artifact_builds) (just think of a super fast Firefox build). It may take a while (10-15 minutes) to download and build Firefox.
+
+There are two commands that you will need. `./mach build` and `./mach run` these can be run together
+each time things change with `./mach build && ./mach run`
+
+* The first time you run the project you will need to use `./mach build` but after the first run you can use `./mach build faster`! The full command looks like this`./mach build faster && ./mach run`
+
+You should now have a fresh copy of nightly open up! You can pass a few properties to it as well,
+such as -P <custom-profile> if you want to use a specific profile
+
+### For Windows Developers
+
+The detailed instructions for setting up your environment to build Firefox for Windows can be found [here](https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/Build_Instructions/Windows_Prerequisites). You need to install the latest `MozBuild` package. You can open a unix-flavor shell by starting:
+
+```
+C:\mozilla-build\start-shell.bat
+```
+
+In the shell, navigate to the debugger.html project folder, and follow the Getting Started instructions as mentioned.
+
+## Configuring Firefox path in the debugger
+
+You will need to copy your new debugger code into firefox each time you change something. By default
+we have the location of firefox to be within the devtools directory.
+
+However if you have firefox installed elsewhere you can update this in `configs/local.json`. There you will find a configuration
+called
+
+```json
+mcPath: './firefox' // expecting firefox to be under `debugger.html/firefox`
+```
+
+You can change this to what works for you!
+
+## Watching for Changes
+
+On to the fun stuff. Each time you change something you will need to copy over your assets, you can
+do this like so!
+
+in npm:
+```
+npm run copy-assets
+```
```

Comment:
lets leave out the `npm` flavor. I think we should expect people to be able to convert btw the two